calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -64,10 +82,48 @@ public void ClearScreen()
             DisplayContent = "0";
         }
 
-        public void ShowPlusOnDisplay()
+        public void ShowPlusOnDisplay(string plus)
+        {
+
+        }
+        public void ShowMultiplyOnDisplay(string multiply)
+        {
+
+        }
+        public void ShowMinusOnDisplay(string minus)
+        {
+
+        }
+        public void ShowdivideOnDisplay(string divide)
+        {
+
+
+        }
```

Comment:
Convert this to a single method with a switch statement. This will make your code less boilerplate.